src/components/Chat/SendMessageButton.tsx

```diff
@@ -1,15 +1,63 @@
 import { IconSend } from '@tabler/icons-react';
+import { ReactNode } from 'react';
+
+import { useTranslation } from 'next-i18next';
 
 import classNames from 'classnames';
 
 import { ConversationsSelectors } from '@/src/store/conversations/conversations.reducers';
 import { useAppSelector } from '@/src/store/hooks';
 import { ModelsSelectors } from '@/src/store/models/models.reducers';
 
+import { Tooltip, TooltipContent, TooltipTrigger } from '../Common/Tooltip';
+
 interface Props {
   handleSend: () => void;
 }
 
+interface SendIconComponentProps {
+  isSendDisabled?: boolean;
+}
+
+interface SendIconTooltipProps {
+  isSendDisabled?: boolean;
+  children: ReactNode;
+}
+
+const SendIconComponent = ({ isSendDisabled }: SendIconComponentProps) => (
+  <span
+    className={classNames(
+      isSendDisabled
+        ? 'text-gray-400 dark:text-gray-600'
```

Comment:
instead of this component you can just use `hover:text-blue-500 disabled:text-gray-400 disabled:dark:text-gray-600`  on line 81 and put IconSend directly on line 92

Reply:
Thank you for pointing at this. 
Implementation changed. Redundant component removed.